src/pcapi/core/users/factories.py

```diff
@@ -23,6 +23,7 @@
 
 
 DEFAULT_PASSWORD = "user@AZERTY123"
+eighteen_years_in_the_past = datetime.now() - relativedelta(years=18, months=4)
```

Comment:
(style) Les constantes sont définies en majuscule.

```suggestion
eighteen_years_in_the_past = datetime.now() - relativedelta(years=18, months=4)
```